category.php

```diff
@@ -1,12 +1,8 @@
 <?php get_header(); ?>
 
-<div class="hero-container">
-  <div class="hero-image" style="background-image: url(<?php echo get_template_directory_uri(); ?>/images/bg/contact.jpg);">
-    <div class="container">
-      <div class="hero-heading">
-        <h1> Projects </h1>
-      </div>
-    </div>
+<div class="hero">
```

Comment:
So, currently it's fine as a partial, but maybe in the future you'll want to include other content that you can't currently think of how to pass in? Sounds like a good time to use a partial.